RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -84,27 +78,28 @@ struct ManageSubscriptionsView: View {
                 background.edgesIgnoringSafeArea(.all)
             }
 
-            ScrollView {
-                VStack {
-                    if self.viewModel.isLoaded {
-                        SubtitleTextView(subtitle: self.viewModel.screen.subtitle)
+            if self.viewModel.isLoaded {
+                List {
+                    if let subtitle = self.viewModel.screen.subtitle {
+                        SubtitleTextView(subtitle: subtitle)
```

Comment:
this is making me wonder if we need the subtitle?

Reply:
What about as header of the buttons section? I think it is fine if the system handles the style since we are setting the color in `SubtitleTextView` to be `textColor`

Reply:
@vegaro Yeah, I think that would be great! I don't want to lose the option to add an additional, optional message, and trying to stuff it above the subscription details feels weird. Will update!